include/translatable-object.php

```diff
@@ -452,4 +452,46 @@ protected function get_objects_with_no_lang_sql( $language_ids, $limit ) {
 			$limit >= 1 ? sprintf( 'LIMIT %d', $limit ) : ''
 		);
 	}
+
+	/**
+	 * Assigns a language to object in mass.
+	 *
+	 * @since 1.2
+	 * @since 3.4 Moved from PLL_Admin_Model class.
+	 *
+	 * @param int[]               $ids  Array of post ids or term ids.
+	 * @param PLL_Language|string $lang Language to assign to the posts or terms.
+	 * @return void
+	 */
+	public function set_language_in_mass( $ids, $lang ) {
+		global $wpdb;
+
+		$lang = $this->get_language( $lang );
+
+		if ( empty( $lang ) ) {
+			return;
+		}
```

Comment:
If it doesn't require too many changes, I suggest to accept only `PLL_Language` and not `string`.

Reply:
🤔 we can in the wizard calls because we have PLL_Language object.
But what about in `PLL_Settings` calls which uses language slug (default_lang option)?
See https://github.com/polylang/polylang/blob/3.3.x/settings/settings.php#L244-L249

Reply:
Done in 8b867bc1174b361bcae57de13b21591d2ad67e0c